spec/system/composer/review_media_unless_trust_level_spec.rb

```diff
@@ -25,14 +27,12 @@
   it "flags a post with an image" do
     topic_page.visit_topic_and_open_composer(topic)
     topic_page.fill_in_composer(" this one has an upload: ")
-    attach_file "file-uploader",
-                "#{Rails.root}/spec/fixtures/images/logo.jpg",
-                make_visible: {
-                  display: "block",
-                  visibility: "visible",
-                  opacity: 1,
-                }
+    attach_file(file_from_fixtures("logo.jpg", "images").path) do
+      composer.click_toolbar_button("upload")
+    end
```

Comment:
The problem is not very clear to me here, I copied what we do in another test, I need to dig this eventually.

Reply:
Playwright has a lower level helper, maybe I should dig using this directly.